Nos/Service/CurrentUser.swift

```diff
@@ -291,3 +292,19 @@ import Dependencies
     }
 }
 // swiftlint:enable type_body_length
+
+extension CurrentUser {
```

Comment:
I'd love to see doc comments on both `func`s below. Would you mind adding some?

Reply:
You bet! Thanks for reminder.